src/main/assets/js/ajaxErrorHandler.ts

```diff
@@ -0,0 +1,11 @@
+import jqXHR = JQuery.jqXHR;
+
+export class AjaxErrorHandler {
+  public static handleError(jqXhr: jqXHR, logMessage: string): void {
+    if(jqXhr.status === 302 && jqXhr?.responseJSON?.url) {
+      window.location.replace(jqXhr.responseJSON.url);
+    } else {
+      console.log(logMessage);
+    }
+  }
+}
```

Comment:
Good way to handle these @agraham01 - cheers for backporting these to other tabs too.